Shared/Samples/Configure electronic navigational charts/ConfigureElectronicNavigationalChartsView.swift

```diff
@@ -0,0 +1,283 @@
+        )
+        
+        // Gets the ENC layer and feature from the identify results.
+        guard let result = identifyResults.first(where: { $0.layerContent is ENCLayer }),
+              let encLayer = result.layerContent as? ENCLayer,
+              let encFeature = result.geoElements.first as? ENCFeature else {
+            return
+        }
+        
+        // Selects the feature using the layer.
+        encLayer.select(encFeature)
+        model.encLayer = encLayer
+        
+        // Sets the callout to display on the feature.
+        let tapLocation = proxy.location(fromScreenPoint: screenPoint)
+        calloutPlacement = .geoElement(encFeature, tapLocation: tapLocation)
+    }
+}
+
+// MARK: Model
+
+private extension ConfigureElectronicNavigationalChartsView {
+    /// The view model for the sample.
+    @MainActor
+    class Model: ObservableObject {
+        /// A map with an oceans basemap.
+        let map: Map = {
+            let map = Map(basemapStyle: .arcGISOceans)
+            map.initialViewpoint = Viewpoint(latitude: -32.5, longitude: 60.95, scale: 67_000)
+            return map
+        }()
+        
+        /// The ENC layer for unselecting the selected feature.
+        var encLayer: ENCLayer?
+        
+        /// A URL to the temporary directory for the generated SENC data files.
+        private let sencDataURL = FileManager.createTemporaryDirectory()
+        
+        deinit {
+            // Resets ENC environment settings when the sample closes.
+            let environmentSettings = ENCEnvironmentSettings.shared
+            ENCEnvironmentSettings.shared.resourceURL = nil
+            ENCEnvironmentSettings.shared.sencDataURL = nil
+            
+            try? FileManager.default.removeItem(at: sencDataURL)
+            
+            let displaySettings = environmentSettings.displaySettings
+            displaySettings.marinerSettings.resetToDefaults()
+            displaySettings.textGroupVisibilitySettings.resetToDefaults()
+            displaySettings.viewingGroupSettings.resetToDefaults()
+        }
+        
+        /// Sets up the ENC exchange set and adds it to the map.
+        func addENCExchangeSet() async throws {
+            // Sets environment settings for loading the dataset.
+            let environmentSettings = ENCEnvironmentSettings.shared
+            environmentSettings.resourceURL = .hydrographyData
+            environmentSettings.sencDataURL = sencDataURL
+            
+            // Creates the exchange set from a local file.
+            let exchangeSet = ENCExchangeSet(fileURLs: [.exchangeSet])
+            try await exchangeSet.load()
+            
+            // Creates layers from the exchange set's datasets and adds them to the map.
+            let encLayers = exchangeSet.datasets.map { dataset in
+                ENCLayer(cell: .init(dataset: dataset))
+            }
+            map.addOperationalLayers(encLayers)
+            await encLayers.load()
```

Comment:
```suggestion
```
Since we don't handle the result of this sequence load, it can be omitted as the layers would be automatically loaded (thus the load method implicitly called) when the view renders.